2-copy-of-code/lesson-13/scripts/amazon.js

```diff
@@ -72,6 +72,7 @@ document.querySelectorAll('.js-add-to-cart')
       const quantitySelector = document.querySelector(
         `.js-quantity-selector-${productId}`
       );
+      const quantity = quantitySelector.value;
```

Comment:
When i try to console.log this I get the error "Uncaught DOMException: Failed to execute 'querySelector' on 'Document': '.js-quantity-selector-${productId}' is not a valid selector."

Reply:
Just realised I was supposed to use template literals instead of quotation marks

Reply:
when i write this line i have void error 
 "Cannot read properties of null (reading 'value')"

Reply:
Document.querySelector: '.js-quantity-selector-15b6fc6f-327a-4ec4-896f-486349e85a3d"' is not a valid selector.
Any idea how to fix it?